tests/expected_files/.copilot/image_build_run.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+# Exit early if something goes wrong
+set -e
+
+# Add commands below to run after all the other build actions have completed
```

Comment:
```suggestion
# Add commands below to run inside the container after all the other buildpacks have been applied
```

Reply:
Resolved in https://github.com/uktrade/copilot-tools/pull/206/commits/e864d163cbb5a7ecf1efe9426e7990b88d5d1dc7.